Src/Application/Score/Age/Commands/UpdateAgeScore/AgeScoreUpdateCommandHandler.cs

```diff
@@ -19,15 +19,15 @@ public async Task<OperationResult<string>> Handle(AgeScoreUpdateCommand request,
 
         if (ageScore is null)
         {
-            return OperationResult<string>.Failed(Resource.RecordNotFound, (int)HttpStatusCode.Created);
+            return OperationResult<string>.Failed(Resource.RecordNotFound, HttpStatusCode.Created);
         }
 
         ageScore.Update(request.FromAge, request.ToAge, request.Score);
 
         _context.AgeScores.Update(ageScore);
         await _context.SaveChangesAsync(cancellationToken);
 
-        return OperationResult<string>.Failed("AgeScore was updated!", (int)HttpStatusCode.Created);
+        return OperationResult<string>.Failed("AgeScore was updated!", HttpStatusCode.Created);
```

Comment:
It should be succeded instead of failed

Reply:
yes i saw it,

it was last repo problem , it was fixed.